webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/TemplatesCard/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@patternfly/react-core';
+import { PencilAltIcon } from '@patternfly/react-icons';
+import { TableComposable, Tr, Tbody, Td } from '@patternfly/react-table';
+import { translate as __ } from '../../../../../../common/I18n';
+import { foremanUrl } from '../../../../../../common/helpers';
+import { useAPI } from '../../../../../../common/hooks/API/APIHooks';
+import SkeletonLoader from '../../../../../common/SkeletonLoader';
+import DefaultLoaderEmptyState from '../../../../DetailsCard/DefaultLoaderEmptyState';
+import CardTemplate from '../../../../Templates/CardItem/CardTemplate';
+import { STATUS } from '../../../../../../constants';
+
+const TemplatesCard = ({
+  hostName,
```

Comment:
```suggestion
  hostname,
```

In Katello I've tried to use the lowercase everywhere for consistency :man_shrugging:

Reply:
But the prop is in camel case?
https://github.com/theforeman/foreman/blob/2c252511913090d3defe58cbc863af5c5920811d/webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/index.js#L122

Reply:
Ohh, we're passing it down from the Slot. Guess we have to leave that as-is then.